ui/lib/apps/Statement/pages/List/index.tsx

```diff
@@ -106,6 +118,9 @@ export default function StatementsOverview() {
                 setQueryOptions({ ...queryOptions, searchText })
               }
             />
+            <Button type="primary" onClick={exportCSV} loading={downloading}>
```

Comment:
maybe we can put it in some extra menu in "..." or bread icon since it is not a frequently used feature. For the same reason, we can place Settings into this new menu.

Reply:
Done.